client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -133,12 +141,21 @@ button {
 		}
 	}
 
+	@include break-large {
+		height: 100%;
+	}
+
 	/**
  	 * Step Container
  	 */
 	.step-container {
 		--color-accent: #117ac9;
 		--color-accent-60: #0e64a5;
+
+		@media ( min-width: 660px ) {
+			display: flex;
+			flex-direction: column;
```

Comment:
@escapemanuele Hiya! I'm curious why do we need this? I made the **ALL** step with a horizontal layout broken...

See p1664411235927349-slack-CRWCHQGUB